pixi.toml

```diff
@@ -101,12 +101,12 @@ pandera = "*"
 pip = "*"
 pre-commit = "*"
 pyarrow = "*"
-pydantic = "~=1.0"
+pydantic = ">=2"
 pyogrio = "*"
 pytest = "*"
 pytest-cov = "*"
 pytest-xdist = "*"
-python = ">=3.9"
+python = ">=3.9, <3.12"
```

Comment:
Do we still have a blocker for 3.12?

If yes, we should also adapt the `pyproject.toml`

Reply:
For context, on main our pixi.lock gives us 3.12 and that seems to be working fine: https://github.com/Deltares/Ribasim/pull/713